### DISCUSSION TITLE
Integrating with API Gateway Like (Ocelot)

### DISCUSSION THREAD

**Discussion by lujamaharjan** [scalar/scalar#3569] [Ideas]
Is there any way to add Scalar with API Gateway like Ocelot.

**Comment by hanspagel** [comment]
Can you give me some more context pls, what API Gateway do you mean?

**Reply by lujamaharjan** [reply]
In a microservice architecture where multiple services expose APIs, configuring Swagger (or similar tools like Swashbuckle) in each service individually can become cumbersome. If I want to centralize API documentation using Swagger at the API Gateway level instead of configuring it for every service, is there an efficient approach to achieve this in case of Scalar? I am particularly taking about Scalar.AspNetCore package.
